localization/english/wc_general_l_english.yml

```diff
@@ -57,3 +57,6 @@
  horde_event_troops:3 "Horde Invaders"
```

Comment:
The last thing, localize them.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/108604297-7a72e100-73c6-11eb-8b22-361d90ef639e.png)

</details>

The localization file is `Warcraft-Guardians-of-Azeroth-2\localization\english\wc_portraits_l_english.yml`

Reply:
Added these in 9ea935592c29d45cc255d859830cfeb9f0ba286e.

<details><summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/78575425/108604602-9d49c900-73b7-11eb-8d36-bc085d62cfb1.png)

</details>